purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -450,6 +451,15 @@ class Purchases internal constructor(
         purchasesOrchestrator.invalidateCustomerInfoCache()
     }
 
+    /**
+     * Used by `RevenueCatUI` to keep track of [PaywallEvent]s.
+     */
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    @JvmSynthetic
```

Comment:
Made this synthetic since this is really only meant for our use in `RevenueCatUI` which uses Kotlin.

Reply:
TIL but makes sense.